udata_front/theme/gouvfr/templates/organization/display.html

```diff
@@ -17,11 +17,14 @@
 {% set bundle = 'organization' %}
 
 {% block breadcrumb %}
+{% cache cache_duration, 'org-breadcrumb', org.id|string, g.lang_code, org.last_modified|string %}
 <li><a href="{{ url_for('organizations.list') }}">{{ _('Organizations') }}</a></li>
 <li class="active" aria-current="page">{{ org.name }}</li>
+{% endcache %}
 {% endblock %}
 
 {% block main_content %}
+{% cache cache_duration, 'org-content', org.id|string, g.lang_code, current_user.slug or 'anonymous', org.last_modified|string %}
```

Comment:
I did not test, but I think you'll need to add a key such as `reuses.page|string` and `datasets.page|string`, to allow for pagination, ex:https://www.data.gouv.fr/fr/organizations/etalab/?datasets_page=2#organization-datasets